subsys/caf/modules/power_manager.c

```diff
@@ -148,19 +150,32 @@ static void system_off_on_error(void)
 	LOG_WRN("System turned off because of unrecoverable error");
 	LOG_PANIC();
 
+#if !defined(CONFIG_SOC_SERIES_NRF54HX)
 	sys_poweroff();
+#else
+	/* On nRF54 series, the poweroff is not supported yet. */
+	LOG_WRN("System suspended, the poweroff is not supported yet on nrf54h20");
+#endif
 }
 
 static void system_off_handler(struct k_work *work)
 {
 	LOG_WRN("System turned off");
 	LOG_PANIC();
+
+#if !defined(CONFIG_SOC_SERIES_NRF54HX)
```

Comment:
Can we introduce Kconfig for clearing the reset reason (Kconfig that depends on `!SOC_SERIES_NRF54HX`)?

Reply:
Similarly for `sys_poweroff` (we can also select either `CONFIG_POWEROFF` or `CONFIG_PM` depending on SoC) ?